docs/CosmWasm/README.md

```diff
@@ -10,4 +10,4 @@ parent:
 
 >CosmWasm is written as a module that can plug into the Cosmos SDK. This means that anyone currently building a blockchain using the Cosmos SDK can quickly and easily add CosmWasm smart contracting support to their chain, without adjusting existing logic.
 
-Read more about writing smart contracts with CosmWasm at their [documentation site](https://docs.cosmwasm.com/docs/1.0/), or visit [the repository](https://github.com/CosmWasm/cosmwasm).
+Read more about writing smart contracts with CosmWasm by reading the [CosmWasm Book]([https://docs.cosmwasm.com/docs/1.0/](https://book.cosmwasm.com/)), or visit [the repository](https://github.com/CosmWasm/cosmwasm).
```

Comment:
The markdown does not look correct. Do you have a reason for linking the book instead of `https://docs.cosmwasm.com/docs/`?

Reply:
because the docs are deprecated, but it seems like someone else made the change already